github/GitRelease.py

```diff
@@ -73,7 +73,6 @@ class GitRelease(CompletableGithubObject):
 
     The reference can be found here
     https://docs.github.com/en/rest/reference/repos#releases
-
```

Comment:
The doc style requires this line, breaks lint test.

Reply:
thanks for correcting!